packages/core/src/app/serialization/rollup/swap-transition.ts

```diff
@@ -0,0 +1,99 @@
+/* Internal Imports */
+import { abi } from '../../../app'
+import { AbiEncodable } from '../../../types'
+
+/**
+ * Creates a TransferNewAccountTx from an encoded TransferNewAccountTx.
+ * @param encoded The encoded TransferNewAccountTx.
+ * @returns the TransferNewAccountTx.
+ */
+const fromEncoded = (encoded: string): AbiSwapTransition => {
```

Comment:
Will this have a name conflict with `swap-tx.ts`, etc. if any of them are imported in the same file?